src/frontend/src/lib/components/send/SendModal.svelte

```diff
@@ -25,7 +27,9 @@
 	let sendProgressStep: string = ProgressStepsSend.INITIALIZATION;
 
 	let steps: WizardSteps;
-	$: steps = sendWizardStepsComplete({ i18n: $i18n });
+	$: steps = isNullish($pageToken)
+		? sendWizardStepsComplete({ i18n: $i18n })
```

Comment:
What does "StepsComplete" mean?

Reply:
we have 3 different lists:

- `sendWizardSteps` --> `SEND` + `REVIEW` + `SENDING`
- `sendWizardStepsWithQrCodeScan` --> the above + `QR CODE`
- `sendWizardStepsComplete` --> the above + `SELECT TOKEN` as additional first step 

if you want I can make `sendWizardSteps` have a parameter that says if it should include the `SELECT TOKEN` step or not, adapt the usage through the code, and then deprecate `sendWizardStepsComplete`

Reply:
OR

Since sendWizardStepsComplete is used only there, i can make in the condition something like

`[ STEP_SEELCT_TOKEN, ... sendWizardStepsWithQrCodeScan]`

Reply:
So "complete" means "all steps"? Maybe just renaming `sendWizardStepsComplete` to `allSendWizardSteps` or something in that direction?

Reply:
yes... makes more sense as name! this PR or another?

Reply:
As you rather like

Reply:
Done with PR https://github.com/dfinity/oisy-wallet/pull/2366